### PR TITLE
Add copyright fork in foo.md

### PR DESCRIPTION
**Pull request by WaximeA**

### CODE REVIEW

**Comment by WaximeA** [comment]
Is this ok ?

File: foo.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Hey to the World !
 =======
 #### We’re trying git at School !
-#### With the fabulous @Nek-  <3.
\ No newline at end of file
+#### With the fabulous @Nek-  <3.
+#### Forked by @WaximeA
```
Comment (PerfectSeiha) [inline foo.md]: Remove your name you didn't do anything on the project :(

**Review by PerfectSeiha — COMMENTED** [review]
Not cool 👎

File: foo.md (hunk 1 above)
Comment (WaximeA) [reply foo.md]: its k dude

Comment (PerfectSeiha) [reply foo.md]: Perfect 🥇

**Pull request merged**